### ISSUE TITLE
Review abstract, intro

### ISSUE THREAD

**Issue by cdeil** [gammapy/gammaskynet-gamma2016#18]
Hi, here's my last round of review:
- [x] Run a spell checker, there's a few typos
- [x] The first paragraph in the abstract has a half sentence at the end that should be removed. ". with details such as ..."
- [x] Introduction - I find the "we" in the introduction a bit weird. "we observed ..." and "we expect to discover". You are using "we" for everyone (all astronomers or humanity). But in the abstract and acknowledgements "we" is us four authors. So I find this a bit confusing and would prefer to avoid using "we" in such a general sense. I think changing to passive voice, taking on more of the role of a passive narrator, doesn't take anything away from the engaging introduction. OK?
- [x] Introduction - You use the abbreviations "HE" and "VHE". If you do, you have to define them by mentioning the energy ranges. I don't care what exaclty you put, you could e.g. use MeV to 100 GeV as "HE" and > 100 GeV as VHE.
- [x] Introduction - The first sentence starts with VHE, but then talks about Fermi, which is usually considered HE range. My suggestion would be start the paper with plain "The field of gamma-ray astronomy is ..." and then to only make the HE / VHE distinction in later sentences if at all (the alternative is just to say that Fermi-LAT does lower-energy gamma-rays (GeV) and ground-based telescopes do very-high-energy gamma-rays (TeV).
- [x] Introduction - I find the last sentence too bold. Concretely I think the "all" is incorrect, we will never expose all gamma-ray catalogs and data (see all of the high-level data products from Fermi alone: http://fermi.gsfc.nasa.gov/ssc/data/access/lat/ ) And I also think the "prepared in a standard format" is misleading ... it suggests we process and offer for download all of the data products from Fermi and the ground-based instruments. Instead I would rephrase the last sentence and say that the goal of gamma-sky.net is to enable people to "explore" and "browse" the gamma-ray sky.

**Comment by vorugantia** [comment]
See my edits [here](https://github.com/gammapy/gammaskynet-gamma2016/commit/ce7e93a5f336db69e770a84faeab8e908726679f). I did everything you mentioned above. Reopen this issue if anything else needs to be changed.
